docs/en/02_Developer_Guides/09_Security/00_Member.md

```diff
@@ -127,6 +127,13 @@ things, you should add appropriate `[api:Permission::checkMember()]` calls to th
 	  }
 	}
 
+## Saved user logins ##
```

Comment:
The contributing guidelines actually say titles should be sentence case, so this is fine here